src/modules/jcms_rest/src/Plugin/rest/resource/AnnualReportListRestResource.php

```diff
@@ -87,8 +87,15 @@ public function getItem(EntityInterface $node) : array {
       }
     }
 
-    // Image is required.
-    $item['image'] = $this->processFieldImage($node->get('field_image'), FALSE, 'thumbnail', TRUE);
+    // Image is required, for version 1.
+    if ($this->acceptVersion < 2) {
+      if ($image = $this->processFieldImage($node->get('field_image'), FALSE, 'thumbnail', TRUE)) {
+        $item['image'] = $image;
+      }
+      else {
+        throw new JCMSNotAcceptableHttpException('This annual report list requires version 2+.');
```

Comment:
Since this is a listing, may be `At least one of the annual report in the list requires version 2+.`?